src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/util/LaunchDarklyFunctionalTestClient.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.iacaseapi.util;
+
+import com.launchdarkly.client.LDClient;
+import com.launchdarkly.client.LDUser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientResponseException;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.UserDetails;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.clients.IdamApi;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.clients.model.idam.UserInfo;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.security.idam.IdamUserDetails;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.security.idam.IdentityManagerResponseException;
+
+@Component
+public class LaunchDarklyFunctionalTestClient {
+
+    @Autowired private IdamApi idamApi;
+
```

Comment:
Could you try to inject `@Autowired private LDClientInterface ldClient` here?
or if it doesn't work change annotation in `CcdScenarioRunnerTest` to something like this:
```
@SpringBootTest(classes = {FeatureToggleConfiguration.class, Application.class})
```
current solution is ok, it is just to keep visibility